### PR TITLE
Density cleanup

### PR DESCRIPTION
**Pull request by Rohesie**
The PR as it should have been initially.
* Cleans some code on density, anchored and opacity variables that were inherited by parents.
* Removes glowshrooms
* Fixes a single case of `use_power` not using the macro.
* Removes some headers because ...reasons?

### CODE REVIEW

**Comment by spookydonut** [comment]
scream unatomic

File: code/game/mecha/mech_bay.dm [hunk 1]
```diff
@@ -1,15 +1,12 @@
...
 	density = TRUE
 	anchored = TRUE
 	icon = 'icons/mecha/mech_bay.dmi'
 	icon_state = "recharge_port"
 
+
 /obj/machinery/computer/mech_bay_power_console
 	name = "Mech Bay Power Control Console"
-	density = TRUE
-	anchored = TRUE
```
Comment (spookydonut) [inline code/game/mecha/mech_bay.dm]: anchored

File: code/game/mecha/mech_fabricator.dm [hunk 2]
```diff
@@ -1,7 +1,3 @@
-/////////////////////////////
-///// Part Fabricator ///////
```
Comment (spookydonut) [inline code/game/mecha/mech_fabricator.dm]: y

File: code/game/mecha/mecha_wreckage.dm [hunk 3]
```diff
@@ -1,8 +1,3 @@
-///////////////////////////////////
-////////  Mecha wreckage   ////////
```
Comment (spookydonut) [inline code/game/mecha/mecha_wreckage.dm]: y

File: code/game/objects/effects/aliens.dm [hunk 4]
```diff
@@ -13,8 +8,6 @@
 	name = "splatter"
 	desc = "It burns! It burns like hygiene!"
 	icon_state = "splatter"
-	density = FALSE
-	opacity = FALSE
```
Comment (spookydonut) [inline code/game/objects/effects/aliens.dm]: opacity

File: code/game/objects/effects/glowshroom.dm [hunk 5]
```diff
@@ -1,77 +1,6 @@
...
-	var/delay = 1200
-	var/floor = 0
-	var/yield = 3
-	var/evolveChance = 2
-	var/lastTick = 0
-	var/spreaded = 1
-
-/obj/effect/glowshroom/single
-
-/obj/effect/glowshroom/Initialize(mapload, ...)
```
Comment (spookydonut) [inline code/game/objects/effects/glowshroom.dm]: undocumented removal

File: code/game/objects/items/reagent_containers/food/snacks/grown.dm [hunk 6]
```diff
@@ -513,17 +513,6 @@
...
 	plantname = "glowshroom"
 
-/obj/item/reagent_container/food/snacks/grown/mushroom/glowshroom/attack_self(mob/user as mob)
-	if(istype(user.loc,/turf/open/space))
-		return
-	var/obj/effect/glowshroom/planted = new /obj/effect/glowshroom(user.loc)
-
-	planted.delay = 50
-	planted.endurance = 100
-	planted.potency = potency
```
Comment (spookydonut) [inline code/game/objects/items/reagent_containers/food/snacks/grown.dm]: undocumented

File: code/game/objects/machinery/OpTable.dm [hunk 7]
```diff
@@ -7,7 +7,7 @@
 	layer = TABLE_LAYER
 	anchored = TRUE
 	resistance_flags = UNACIDABLE
-	use_power = 1
+	use_power = IDLE_POWER_USE
```
Comment (spookydonut) [inline code/game/objects/machinery/OpTable.dm]: unatomic

Comment (Rohesie) [reply code/game/objects/machinery/OpTable.dm]: Do you want it done in a separate PR?

**Comment by Rohesie** [comment]
Documentation done.

**Pull request closed without merging**